db/projects/loopring/project.yaml

```diff
@@ -0,0 +1,32 @@
+name: Loopring
```

Comment:
Here is logo of Loopring from official websites:
https://loopring.org/images/logo.svg

Reply:
asset-lrc-logo-full.svg was added.